observation/rfi_scan.py

```diff
@@ -34,9 +33,7 @@
 el_end = 27.1  # 15.1
 scan_spacing = 5.0 # 6.0
 num_scans = 3
-scan_duration = 180.
-if opts.long is not None:
-    scan_duration = float(opts.long)
+scan_duration = float(opts.scan-duration)
```

Comment:
No need to cast to float -- it's already a float. Plus `scan-duration` would have needed an underscore. Rather do: `scan_duration = opts.scan_duration`.

Reply:
Done